src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -145,15 +168,22 @@ var styles = StyleSheet.create({
     flex: 1
   },
   backButton: {
+    padding: 5,
+    right: 10,
+    top: 100
+  },
+  generalButtonStyle: {
     alignItems: 'center',
     backgroundColor: colors.gray60,
     borderRadius: 50,
     justifyContent: 'center',
-    padding: 5,
     position: 'absolute',
-    right: 10,
-    top: 100,
     zIndex: 1
+  },
+  screenShotButton: {
+    bottom: 100,
+    padding: 15,
+    right: 10
```

Comment:
could we need `normalize` here?

and i would suggest to place the button closer to the bottom like in the figmas:

|iPhone 6s|figma|
|---|---|
|![IMG_7AC738AB7ACC-1](https://user-images.githubusercontent.com/1942953/178775582-e32c0a4a-a245-4784-b534-84a606b13162.jpeg)|![Bildschirmfoto 2022-07-13 um 17 41 59](https://user-images.githubusercontent.com/1942953/178775023-947132a0-6e12-41ea-8e72-541b14a35401.png)|

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/576/commits/4e0bb58d3058f0c7989cc2ec9fbbc1f735072db3

Reply:
|iPhone 13 Pro Max|iPhone 7|
|--|--|
![IMG_0241](https://user-images.githubusercontent.com/11755668/179163044-f8dfdcf0-0a7c-40dc-9ecd-9923e6202d1c.PNG) | ![IMG_0002](https://user-images.githubusercontent.com/11755668/179163063-ec61fd59-670d-4cc3-bc53-7ea2956a3231.PNG)

Reply:
could we add some transparency to the background gray, like in the figmas?

Reply:
![IMG_EBC1EF28AFAA-1](https://user-images.githubusercontent.com/11755668/179209894-d207bb08-9155-45bf-8046-8b611189e916.jpeg)

https://github.com/ikuseiGmbH/smart-village-app-app/pull/576/commits/bf6731dc99735ec509d8a5c91bb056e2280003fc

Reply:
thanks, now please make it white like in the figmas